### ISSUE TITLE
[FEATURE] iOS support

### ISSUE THREAD

**Issue by sdatkinson** [sdatkinson/NeuralAmpModelerPlugin#261]
Issue to track support for iOS.

**Comment by sdatkinson** [comment]
Requires #114

**Comment by sdatkinson** [comment]
Just built it onto my iPad Air (4th generation) today. A few nits, but it ran fine :)

**Comment by demym** [comment]
@sdatkinson did you try to run it on iPad using an external soundcard ? As you may know, to me it crashes that way

**Comment by sdatkinson** [comment]
@demym  

> @sdatkinson did you try to run it on iPad using an external soundcard ? As you may know, to me it crashes that way

Yep. Used a Focusrite Scarlett and played the standalone, routing my instrument through input 1 (via DI box) and listened via the interface's headphones output.

I don't recall if I had to do any configuring to use the interface originally, but I've used it to record on some version of ProTools I think it was, if I'm not mistaken.

**Comment by mariuslacatus** [comment]
iOS support would absolutely be great. I'll try my hand at trying to build it as well. Any useful tips or gotchas for building it for iOS?

**Comment by bobbyj8866** [comment]
I would buy an Auv3 version of NAM for iOS. That would sell. And support further development methinks.

**Comment by Offroaders123** [comment]
I am looking to locally compile the NAM plugin to 'iOS-App with AUv3' so I can run it as a dev build on my iPad. I am getting errors in Xcode with trying to resolve paths to certain dependencies. Is it something I didn't configure correctly?

<img width="538" alt="Compilation Issue Log" src="https://github.com/user-attachments/assets/512dcfc6-9a3c-4cb6-959b-ae94ec24cd5a" />

My thoughts are that it has something to do with how the Groups are defined inside of the `.xcodeworkspace` or `.xcodeproj` file. I am starting to get into Xcode development, so I'm not too familiar with things yet.

I am however able to locate the missing `resample.cpp` file inside of the folder itself, it just seems that Xcode can't resolve where it is.

Thank you! 🎸

**Comment by sdatkinson** [comment]
Ah. You probably need to add these files to the XCode workspace. I might have missed doing that with some files.

It might feel a little odd but XCode kind of has its own project file organization that's independent from how the files exist on the computer filesystem. I personally don't find it super-helpful, but c'est la vie. You get used to it.

Feel free to PR me a change that adds them.

Feel free also to PR something that changes the CI build to try making the iOS AUv3--curious if we can just keep that working.

**Comment by Offroaders123** [comment]
Do I drag it onto the Workspace in Xcode, is there a specific group it has to go under to fix it?
